keyvault/data-plane/azure-keyvault-core/pom.xml

```diff
@@ -12,6 +12,7 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
+  <groupId>com.microsoft.azure</groupId>
```

Comment:
We should default the other properties like versioning and developer, etc in the parent POM as well to avoid needing them in all the individual projects.

Reply:
It might help with some local views of the project but it adds more harm then good in my opinion as it enables more projects to stay in sync, in particular for the version number, the fewer places we have to edit that the better. So I still suggest that a number of common properties should be lifted into one of the parent POM's in the structure. Of course that can happen as a clean-up PR and not block this PR.

Reply:
I'm fine to centralise versions where it makes sense. I would rather not centralise group I'd for now though. I would say this should be done subsequently so that it does not hold merging. We should get this merged ASAP

Reply:
I agree.